great_expectations/expectations/metrics/map_metric_provider/spark_methods.py

```diff
@@ -427,7 +428,7 @@ def _spark_column_pair_map_condition_filtered_row_count(
 
 def _spark_multicolumn_map_condition_values(
     cls,
-    execution_engine: SqlAlchemyExecutionEngine,
+    execution_engine: SparkDFExecutionEngine,
```

Comment:
I'm pretty sure this was a mistake in the previous code.

Reply:
@abegong You definitely caught a typo (a subtle one, because it was in the type hint, causing no crashes).  Thanks.